Packs/CommonScripts/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Common Scripts",
     "description": "Frequently used scripts pack.",
     "support": "xsoar",
-    "currentVersion": "1.2.36",
+    "currentVersion": "1.3.0",
```

Comment:
should be `1.2.37`. this is a bug fix, not an enhancement

Reply:
Done automaticaly by the sdk (did `-u minor`) I'll fix it